docs/modules/servers/pages/postgres/objectives.adoc

```diff
@@ -0,0 +1,22 @@
+= Distributed James Server &mdash; Objectives and motivation
+:navtitle: Objectives and motivation
+
+From the outstanding advantages of a distributed mail system, such as scalability and enhancement,
+this project aims to implement a backend database version using Postgres,
+
+Primary Objectives:
+
+* Provide more options: The current James Distributed server uses Cassandra as the backend database.
+  This project aims to provide an alternative to Cassandra, using Postgres as the backend database.
+  The choice of version depends on the operator's decision.
+* Replace the `jpa-app` variant: The `jpa-app` variant is a simple version of James that uses JPA to store data.
```

Comment:
Not aimed to replace jpa, people will still use jpa in the community, as some use other sql dbs than postgresql with the jpa version. I disagree with that statement

It's more proposing I think an alternative to jpa with a popular db, Postgresql, that would be reactive and better optimized.